### ISSUE TITLE
[web] [Regression] Web does not run in new rnv project using flexn/template

### ISSUE THREAD

**Issue by pauliusguzas** [flexn-io/flexn#23]
rnv version: 0.36.1-canary.2
flexn template version: 0.16.0

How to repro: 
1. Create new project using `rnv new` and select @flexn/template
2. Do `rnv run -p web`

```
✔ Executing: npx next dev --port 8080

ready - started server on 0.0.0.0:8080, url: http://localhost:8080
Found custom dir:/Users/pauliusguzas/Desktop/flexnTemplate/src/pages
info  - Disabled SWC as replacement for Babel because of custom Babel configuration "babel.config.js" https://nextjs.org/docs/messages/swc-disabled
info  - automatically enabled Fast Refresh for 1 custom loader
wait  - compiling...
error - ./node_modules/react-native-media-query/utils/create-declaration-block.js:1:0
Module not found: Can't resolve 'react-native-web/dist/exports/StyleSheet/createReactDOMStyle'

Import trace for requested module:
./node_modules/react-native-media-query/create-stylesheet/index.web.js
./node_modules/react-native-media-query/index.web.js
./src/config.tsx
./src/pages/_app.tsx

https://nextjs.org/docs/messages/module-not-found
info  - automatically enabled Fast Refresh for 1 custom loader
info  - automatically enabled Fast Refresh for 1 custom loader
info  - Using external babel configuration from /Users/pauliusguzas/Desktop/flexnTemplate/babel.config.js
info  - automatically enabled Fast Refresh for 1 custom loader
info  - automatically enabled Fast Refresh for 1 custom loader
info  - automatically enabled Fast Refresh for 1 custom loader
info  - automatically enabled Fast Refresh for 1 custom loader
info  - automatically enabled Fast Refresh for 1 custom loader
info  - automatically enabled Fast Refresh for 1 custom loader
info  - automatically enabled Fast Refresh for 1 custom loader
info  - automatically enabled Fast Refresh for 1 custom loader
wait  - compiling /_error (client and server)...
wait  - compiling...
info  - automatically enabled Fast Refresh for 1 custom loader
```

**Comment by aurimasmi** [comment]
Submitted PR https://github.com/kasinskas/react-native-media-query/pull/12

**Comment by pauliusguzas** [comment]
Fixed on @flexn/template@0.18.0-alpha.8
